twit.css

```diff
@@ -0,0 +1,127 @@
+body {
```

Comment:
Overall not bad. There are few feedbacks with the page: 
1. You could use different font that is closer to what actual Twitter page has. 
2. Elements could be centered more on both blue and white sides. 
3. That bird background is good. It is centered in the middle of the page which is what Twitter page has. 
4. Good use of icons for blue side. 
5. Page does not respond well when browser is resized. 
Other than above feedbacks, page looks fine and html elements are well organized with divs. Keep trying new CSS styles and you will create cool looking pages in no time.